longclaw/shipping/utils.py

```diff
@@ -8,28 +10,57 @@ class InvalidShippingRate(Exception):
 class InvalidShippingCountry(Exception):
     pass
 
-def get_shipping_cost(settings, country_code=None, name=None):
+def get_shipping_cost(settings, country_code=None, name=None, basket_id=None, destination=None):
```

Comment:
Perhaps `country_code` and `destination` can be merged? Is there a need to keep both (bear in mind I don't think the user base is quite large enough at the moment to be too worried about backwards compatibility)

Reply:
Seems so to me.  I can't see a case where it would be preferable to only ask for the shipping country.  You have to have the complete shipping address no matter what eventually.

Reply:
If you're okay with it I would prefer to keep backwards compatibility.  Maybe a follow up cleanup could remove it but it helps me to know all these tests still pass as originally intended. I can also see usefulness if the site doesn't implement any calculated rate providers and not needing the destination address all the time